htdocs/compta/facture/card.php

```diff
@@ -5780,8 +5780,7 @@ function js_recalculate_revenuestamp(){
 				}
 				// For down payment invoice (deposit)
 				if ($object->type == Facture::TYPE_DEPOSIT && $usercancreate && $object->statut > Facture::STATUS_DRAFT && empty($discount->id)) {
-					if (price2num($object->total_ttc, 'MT') == price2num($sumofpaymentall, 'MT') || ($object->type == Facture::STATUS_ABANDONED && in_array($object->close_code, array('bankcharge', 'discount_vat', 'other')))) {
-						// We can close a down payment only if paid amount is same than amount of down payment (by definition)
+					if (price2num($object->total_ttc, 'MT') == price2num($sumofpaymentall, 'MT') || !empty($conf->global->DEPOSIT_AS_CREDIT_AVAILABLE_EVEN_UNPAID) || ($object->type == Facture::STATUS_ABANDONED && in_array($object->close_code, array('bankcharge', 'discount_vat', 'other')))) {
```

Comment:
Use 
getDolGlobalInt('DEPOSIT_AS_CREDIT_AVAILABLE_EVEN_UNPAID')
instead of 
!empty($conf->global->DEPOSIT_AS_CREDIT_AVAILABLE_EVEN_UNPAID)